app/lib/filter-components.js

```diff
@@ -1,20 +1,20 @@
-var frameworkComponents = [];
+const frameworkComponents = [];
 
 function getCfComponents( data ) {
-  data.forEach( function(el, i) {
-    var name;
+  data.forEach( function( el, i ) {
+    let name;
     if ( el.type === 'dir' ) {
```

Comment:
Now that `capital-framework` is in the `packages/` dir it's getting pulled into this list of installable components. Maybe do:

```suggestion
    if ( el.type === 'dir' && el.name !== 'capital-framework' ) {
```